assets/js/modules/analytics/components/common/PropertySelectIncludingGA4.js

```diff
@@ -91,7 +93,11 @@ export default function PropertySelectIncludingGA4() {
 				return;
 			}
 
-			trackEvent( 'analytics_setup', 'property_change', newPropertyID );
+			const action =
+				newPropertyID === PROPERTY_CREATE
+					? 'change_property_new'
+					: 'change_property';
+			trackEvent( `${ viewContext }_analytics`, action, 'ua' );
```

Comment:
The event label here should be either `ua` or `ga4` based on the currently selected property. If the `PROPERTY_CREATE` is selected, then the event label should be `ua`.

Reply:
I _think_ https://github.com/google/site-kit-wp/pull/4321/commits/12a6cd6adbdcf42615ca4c16ff6a03e0878994f6 is right, can you double check please @eugene-manuilov